Mathlib/RingTheory/Smooth/Kaehler.lean

```diff
@@ -3,7 +3,10 @@ Copyright (c) 2024 Andrew Yang. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Andrew Yang
 -/
-import Mathlib.RingTheory.Kaehler.Basic
+import Mathlib.RingTheory.Kaehler.CotangentComplex
+import Mathlib.RingTheory.Smooth.Basic
+import Mathlib.Algebra.Module.Projective
+import Mathlib.Tactic.StacksAttribute
```

Comment:
```suggestion
```

Reply:
This import is needed for the stacks tag. I added a noshake entry.